.github/workflows/build.yml

```diff
@@ -0,0 +1,128 @@
+name: build
+
+on: ['push']
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Shellcheck
+        uses: reviewdog/action-shellcheck@v1
+        with:
+          github_token: ${{ secrets.github_token }}
+          reporter: github-pr-review
+          path: '.'
+          pattern: '*.sh'
+          exclude: './.git/*'
+
+  ndm-daemonset:
+    runs-on: ubuntu-latest
+    needs: lint
+    env:
+      IMAGE_ORG: 'raspbernetes'
```

Comment:
This can be removed now as the image can be pushed to openebs docker hub.

Reply:
Perfect will update this now 👍